bedrock/products/templates/products/vpn/download.html

```diff
@@ -37,121 +39,219 @@
 
 {% block content %}
 <main>
-  <div class="mzp-l-content">
-    <header class="vpn-download-header">
-      <h1>{{ self.page_title_full() }}</h1>
-      {% if block_download %}
+  <section class="vpn-downloads">
+    <div class="mzp-l-content">
+      <header class="vpn-download-header">
+        {% if block_download %}
+        <h1>{{ self.page_title_full() }}</h1>
         <p>{{ ftl('vpn-download-not-available-in-country') }}</p>
-      {% else %}
-        <p>{{ ftl('vpn-download-sub-heading', devices=connect_devices) }}</p>
-      {% endif %}
-    </header>
 
-    {% if not block_download %}
-      <ul class="vpn-download-list">
-        {% call picto(
-          base_el='li',
-          title=ftl('vpn-download-for-windows'),
-          image=resp_img(
-            url='img/products/vpn/download/platform-windows-black.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'width': '100',
-              'loading': 'lazy'
-            }
-          ),
-          body=True,
-        ) %}
-          <p>{{ ftl('vpn-download-for-windows-requirements') }}</p>
-          <a class="mzp-c-button mzp-t-secondary" href="{{ windows_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Windows)">
-            {{ ftl('vpn-download-button-copy') }}
-          </a>
-        {% endcall %}
+        {% else %}
 
-        {% call picto(
-          base_el='li',
-          title=ftl('vpn-download-for-mac'),
-          image=resp_img(
-            url='img/products/vpn/download/platform-mac-black.svg',
-            optional_attributes={
-              'class': 'mzp-c-picto-image',
-              'width': '100',
-              'loading': 'lazy'
-            }
-          ),
-          body=True,
+        {% call split(
+        block_class='mzp-l-split-body-wide mzp-l-split-center-on-sm-md',
```

Comment:
Issue (potentially blocking): at smaller viewports the header graphic can sometimes overlap with the copy. This might only be an issue with longer words, such as for the German locale.

Maybe we could try and switch to single column at an earlier break point?

![image](https://user-images.githubusercontent.com/400117/231711179-f67d7ea0-1239-4a23-8552-cee5e5e677d8.png)